apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -108,14 +108,14 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
         const settingValues = {
             role: {
                 claim: getDefaultDropDownValue(dropDownOptions, values.get("roleAttribute")),
-                includeUserDomain: values.get("role").includes("includeUserDomain"),
+                includeUserDomain: values.get("role")?.includes("includeUserDomain"),
```

Comment:
Already fixed with https://github.com/wso2/identity-apps/pull/1947. Better to check for conflicts.